base/distributed/cluster.jl

```diff
@@ -167,8 +175,6 @@ function start_worker(out::IO, cookie::AbstractString)
     print(out, LPROC.bind_addr)
     print(out, '\n')
     flush(out)
-    # close STDIN; workers will not use it
```

Comment:
Have temporarily commented out this line as this test was failing - https://github.com/JuliaLang/julia/blob/96e430199b7104134e79a0daffe6c97caca84c90/test/show.jl#L421-L425

However, I believe we should close STDIN on the workers and change the test. Any reasons to keep STDIN open on the workers?